### ISSUE TITLE
FYI github broke this

### ISSUE THREAD

**Issue by bootstraponline** [jackwilsdon/original-github-navigation-bar-color#2]
> document.querySelector('div.header.header-dark.header-logged-in').classList.remove('header-dark');

```
VM181:1 Uncaught TypeError: Cannot read property 'classList' of null
    at <anonymous>:1:66
```

see https://github.com/victor-li/github-white-navbar for the fix

**Comment by turbopixel** [comment]
Yes, Github fix it... Here is my solution:

```
       const header = document.getElementsByClassName('header');
	if (header.length) {
	  header[0].className += " header-dark";
	}
```

Now header-dark is the new hell

**Comment by jackwilsdon** [comment]
Fixed in 1f19a5a1b8588ab54087503bb530e8bd33389609 (which has been released as 1.1.0), although I had to remove toggle support because it caused a visible flash on page load.

I'm in the process of updating the UserScript too.

I've uploaded the new version to the Chrome Web Store. 🎉
